libs/wallet-ui/src/hooks/use-import-network.tsx

```diff
@@ -56,6 +57,7 @@ export function useImportNetwork() {
       } catch (err: unknown) {
         setError(`${err}`)
         setStatus(FormStatus.Error)
+        logger.error(err)
```

Comment:
it would be good to add logger as a dependency in the callback, otherwise it LGTM

Reply:
true, added, thanks!